src/main/java/carpet/mixins/PlayerList_fakePlayersMixin.java

```diff
@@ -41,7 +42,9 @@ private ServerGamePacketListenerImpl replaceNetworkHandler(MinecraftServer serve
     {
         if (playerIn instanceof EntityPlayerMPFake fake)
         {
-            return new NetHandlerPlayServerFake(this.server, clientConnection, fake, cookie);
+            var handler = new NetHandlerPlayServerFake(this.server, clientConnection, fake, cookie);
+            FakePlayerManager.connections.add(handler);
```

Comment:
where's the corresponding code for removing the handler? Adding without deleting might cause a slow memory leak in servers that run for a long time.

Reply:
https://github.com/gnembon/fabric-carpet/pull/1847/files/303e099b5885fd4c77285e0c4ea87a41a45ca50b#diff-c59c3e90cb5d221f2a8191ae3e51d58741a8b3a4c48d48773859d4fa7b4f25d8R11

Reply:
handlers from removed fake players will also be ticked here, which does not seem correct https://github.com/gnembon/fabric-carpet/pull/1847/files#diff-b8fad85a8571843b58c03460fb7bf66949628f5a369d0ae14d2b97834199e0adR21

Reply:
> https://github.com/gnembon/fabric-carpet/pull/1847/files/303e099b5885fd4c77285e0c4ea87a41a45ca50b#diff-c59c3e90cb5d221f2a8191ae3e51d58741a8b3a4c48d48773859d4fa7b4f25d8R11

that's not a correct implentation since `onServerClosed` only gets invoked once per server lifecycle